web/package.json

```diff
@@ -50,9 +51,11 @@
     "rebass": "^1.0.4",
     "redux": "^3.7.2",
     "redux-form": "^7.2.1",
+    "redux-thunk": "^2.2.0",
```

Comment:
:tada: I love redux-thunk.

Reply:
agreed -- i think it's the simplest and most understandable of the lot